include/fmt/printf.h

```diff
@@ -474,8 +474,13 @@ OutputIt basic_printf_context<OutputIt, Char>::format() {
   const Char* end = parse_ctx_.end();
   auto it = start;
   while (it != end) {
+    if (!detail::find<false, Char>(it, end, '%', it)) {
+      it = end;
```

Comment:
I think this should be `break` and the if below should be removed because if we didn't find `%` there is no point in searching again.

Reply:
Thanks for the catch.